guide/inheritance-wallet/landing-page.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: guide
+title: Inheritance wallet
+description: An in-depth UX reference design for a multi-key bitcoin wallet with inheritance features designed for families.
+nav_order: 6
+has_children: true
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/header.png
+image_base: /assets/images/guide/inheritance-wallet/
```

Comment:
Can delete the image_base line if you don't use the `_image-gallery` partial anywhere. You can also remove the `images_creation-wallet-basics:` object right underneath if you don't use us.